### ISSUE TITLE
Training sets for IIIT5k, SVT, IC13, IC15, SVTP, and CUTE

### ISSUE THREAD

**Issue by dparres** [HCIILAB/Scene-Text-Recognition-Recommendations#8]
Hello!

Is it possible to obtain the training sets of the datasets: IIIT5k, SVT, IC13, IC15, SVTP, and CUTE? I can see the links to Baidu for the text sets, but I need the training sets.

Best regards and thank you very much,

Daniel

**Comment by Mountchicken** [comment]
Hi @dparres 
We don't have the training set of these 6 benchmarks either. However, you can simply download them through the dataset preparer in MMOCR. Check this out  https://mmocr.readthedocs.io/en/dev-1.x/user_guides/data_prepare/dataset_preparer.html#dataset-preparer-beta

**Comment by dparres** [comment]
Thank you very much! I have a doubt, for the SVTP case there is no train set, so I should use the SVT train set and evaluate with SVTP? Or with which dataset should I train to evaluate on SVTP?

**Comment by dparres** [comment]
And what train set is used for CUTE? Or when in the paper it says (Syn+Benchmark) is trained using the synthesized data and ALL train sets from the STR databases (IIT5k SVT IC13 IC15) at the same time? To later evaluate individually on each STR test set (IIT5k SVT IC13 IC15 SVTP CUTE)?

**Comment by Mountchicken** [comment]
@dparres 
In scene text recognition, there are two mainstream paradigms. The first one is to train the model solely on synthetic datasets (e.g. MJ and ST) and evaluate the six benchmarks (We don't use the training set of these six benchmarks during training, but only use their test set for benchmark). The second one is to train the model on some [real datasets](https://github.com/Mountchicken/Union14M#3-union14m-dataset). Anyway, neither of these two paradigms will use the training set of the six benchmarks.

**Comment by dparres** [comment]
Agreed! Thank you very much!

Best regards,
Daniel
